Packs/CommonTypes/IndicatorFields/indicatorfield-STIX_ID.json

```diff
@@ -1,5 +1,5 @@
 {
-    "associatedToAll": false,
+    "associatedToAll": true,
```

Comment:
You changed this field in the mutex PR, lets choose one.

Reply:
Its "true" in both, because this PR is forked from the mutex one which should be merged first. It's on purpose